### ISSUE TITLE
LibreOffice templates

### ISSUE THREAD

**Issue by rskikuli** [uradotdesign/task-management#78]
We need to design template with Ura branding for LibreOffice writer and Impress.

**Comment by rskikuli** [comment]
Due to planned presence of Ura and Identihub in different events and conferences I would propose to work on this relatively soon. What do you think @elioqoshi @AnXh3L0 ?

**Comment by elioqoshi** [comment]
Too much of backlog right now and Anxhelo and me have no breathing space for the upcoming weeks. This should be a good task for a new designer who could work with us.

**Comment by rskikuli** [comment]
True. As.mentioned internally meetings with potential desingers will follow. Updates will be published here.

**Comment by rskikuli** [comment]
Check https://twitter.com/libreoffice/status/1009076674573500418

**Comment by elioqoshi** [comment]
Let's circle back to this once we have some updated branding of Ura. Our current white and blue palette is not enough for an extensive visual identity.
